tests/model_inheritance/tests.py

```diff
@@ -149,6 +150,12 @@ def test_custompk_m2m(self):
         # accidentally found).
         self.assertSequenceEqual(s.titles.all(), [])
 
+    def test_create_diamond_mti(self):
+        with self.assertNumQueries(4):
```

Comment:
This may be a question more for felix & charettes but do we care about the num of queries? 🤔 The main issue is that it creates without crashing ✓

Reply:
As this is the only test covering the save of multi table inheritance I think asserting queries will be relevant.

Reply:
> Agreed with @Akash-Kumar-Sen, if we can address this problem minimal number of queries (one for each base) I don't see harm in asserting against it.

Ah so it's 1 query per base? I was originally thinking it's a brittle test but if it lines up with the algorithm then that's fine 👍

Worth commenting to that effect? 🤔 Or even a constant to replace the magic number?

Reply:
> Worth commenting to that effect? 🤔 Or even a constant to replace the magic number?
Yes, a comment explaining the fact that it is one query per base seems relevant.